### DISCUSSION TITLE
Datepicker - month view shifts when selecting date ranges across months

### DISCUSSION THREAD

**Discussion by Cynthiafan** [primefaces/community#3057] [PrimeVue]
## Description

I have a datepicker:
```
<DatePicker v-model="date" :numberOfMonths="2" selectionMode="range" />
```

When I try to select a date range across months, the first month's view always jumps to match the month of my second selection. 
For example, if the initially displayed months are October and November, and I select October 1st as the start date and November 1st as the end date, the displayed months will shift to November and December.

I believe this behavior feels unnatural to users, as they would expect to see both months displayed side by side without needing to click the prev month button again.

## Reproduce

https://stackblitz.com/edit/cxivyk?file=src%2FApp.vue

https://github.com/user-attachments/assets/54f9e144-62c1-4f93-8ffb-faa8be94638c

## Suggestion

Would it be possible to add a prop to control and prevent this behavior from occurring?
https://github.com/primefaces/primevue/blob/master/packages/primevue/src/datepicker/DatePicker.vue#L1157-L1158

**Comment by anexis-fabianamhof** [comment]
+1

**Comment by ehacihalil** [comment]
+1 

Just wondering if there's a recommended way to handle or bypass this behavior for now? Even a workaround or a bit of a hack would be super helpful

**Comment by thatsenam** [comment]
https://github.com/primefaces/primevue/issues/7691 
@Cynthiafan  this issue is resolved
